addon/components/primitives/frost-list-item-large-icon-text.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember'
```

Comment:
Can this component just be done using a class modifier on the `icon-text` primitive?